internal/config/config.go

```diff
@@ -169,6 +169,22 @@ type Instrumentation struct {
 	NetworkCapture bool `yaml:"networkCapture,omitempty" json:"networkCapture"`
 }
 
+// Suite represents the general test suite configuration.
+type Suite struct {
```

Comment:
But we don't have a general test suite configuration. You can already see that with attributes like `Browser`, `PreExec` or `Shard`, which are not applicable on all frameworks.

Reply:
I'm trying to find a a good way to manage suite and avoid cycle importing. Is it a good choice to rename it to `CypressSuite`?

Reply:
moved it to a separate package `internal/cypress/suite`. Should solve the import cycle 🤔